src/SwedbankPay.Sdk/Payments/MobilePayPayments/MobilePayPayment.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Swedbankpay.Sdk.Payments.MobilePayPayments;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.MobilePayPayments
+{
+    public class MobilePayPayment
+    {
+        private MobilePayPayment(MobilePayPaymentResponse paymentResponse, HttpClient client)
+        {
+            PaymentResponse = paymentResponse.Payment;
+            var operations = new MobilePayPaymentOperations(paymentResponse.Operations, client);
+            Operations = operations;
+        }
+
+        public MobilePayPaymentOperations Operations { get; }
+
+        public PaymentResponseObject PaymentResponse { get; }
+
+
+        internal static async Task<MobilePayPayment> Create(MobilePayPaymentRequest paymentRequest,
```

Comment:
The `Create` method should be exposed on the `MobilePayPaymentsResource` directly under `SwedbankPayClient`, no?

Reply:
Isn't this method exposed through `MobilePayPaymentsResource` under `SwedbankPayClient`?

Reply:
It may be, but I think the implementation is in the wrong place. Is this how `Create` is implemented in all other payment instruments as well?